app/main/helpers/search_summary_manifest.yml

```diff
@@ -85,6 +85,28 @@
   filtersPreposition:
   filterRules:
   conjunction: and
+
+
+-
+  id: Supplier type
+  labelPreposition: of type
```

Comment:
I think if you want to suggest a better preposition thing then that's cool. Basically the more filters there are, the closer to impossible it becomes to have the thing make a sensible sentence. And I don't think it will be possible to add punctuation that consistently works with this mechanism, although maybe it would be and maybe that could help.

After this sprint we need to look at whether this summary thing is right (placeholder ticket @ <https://trello.com/c/SpHlwgBj/440-search-summary-refactoring>), short run we need _something_ so that every filter shows up in the summary.